src/main/java/com/bootcamp/demo/controller/CardController.java

```diff
@@ -1,24 +1,37 @@
 package com.bootcamp.demo.controller;
 
+import com.bootcamp.demo.model.Card;
 import com.bootcamp.demo.service.CardService;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.WriteResult;
+import com.google.firebase.cloud.FirestoreClient;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
 
-import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import java.util.List;
 
 @RestController
-@RequestMapping(value = "/card", produces = APPLICATION_JSON_VALUE )
+@RequestMapping("/card")
 public class CardController {
-    private final CardService cardService;
+    private Firestore firestoreDB;
+    private CardService cardService;
 
+    @Autowired
     public CardController(CardService cardService) {
         this.cardService = cardService;
     }
 
-    @DeleteMapping(path="{id}")
-    ResponseEntity<String> removeCard(@PathVariable String id){
-      this.cardService.removeCard(id);
-      return new ResponseEntity<>("Successfully deleted card "+ id, HttpStatus.OK);
+    @DeleteMapping(value = "/{cardNumber}")
+    public String removeCard(@PathVariable String cardNumber){
+        this.firestoreDB = FirestoreClient.getFirestore();
+        ApiFuture<WriteResult> writeResult = firestoreDB.collection("cards").document(cardNumber).delete();
+        return "Successfully deleted " + cardNumber;
+    }
+
+    @GetMapping("/getAllCards")
+    @ResponseBody
```

Comment:
The `@ResponseBody` annotation is not needed. It is already included in the `@RestController` annotation used for this controller.

Reply:
Fixed